proprietary/design-tokens/src/component/utrecht/calendar.tokens.json

```diff
@@ -98,24 +110,25 @@
         }
       },
       "navigation": {
-        "background-color": {
-          "value": "{utrecht.color.black}"
-        },
+        "background-color": {},
         "color": {
-          "value": "{utrecht.color.white}"
+          "value": "{utrecht.color.blue.30}"
         },
         "padding-block-start": {
-          "value": "20px"
+          "value": "2px"
         },
         "padding-block-end": {
-          "value": "20px"
+          "value": "2px"
         },
         "label": {
           "min-inline-size": {
             "value": "200px"
           },
           "color": {
-            "value": "{utrecht.color.white}"
+            "value": "{utrecht.color.black}"
+          },
+          "min-font-size": {
```

Comment:
What's `min-font-size`?

Reply:
I actually haven't the foggiest. Refactored that design token now :D